### PR TITLE
AFP-280 Initial commit

### PR DESCRIPTION
**Pull request by qburst-aparna**
This is an initial draft of the root level readme page generation

### CODE REVIEW

**Comment by Noviny** [comment]
🎉 Fantastic. Couple of small things to change:

Can we make this the left-most card, not the right-most card? Also, probably not call it readme, though I don't know what to call it.

Checked with Nadia as well and she's 👍

**Comment by Noviny** [comment]
image

This starts looking weird on smaller screens. We probably want the boxes to start stacking once there are more than 3. Would it be possible to add this to this PR? If not, we can create a second ticket for it.

Also, the text should probably be more generic (even just for get started) - this text for example wouldn't make sense in the context of Tangerine, which will be the first place to use this.

File: packages/website/pages/index.tsx [hunk 1]
```diff
@@ -49,6 +49,16 @@ class HomePage extends React.Component {
     }
   }, 100);
 
+  getReadmePanelProps = () => {
+    return {
+      IconComponent: MediaDocIcon,
+      label: 'Get Started',
+      color: colors.R400,
+      description: 'Everything you need to get up and running',
+      imgSrc: '/static/code.png',
```
Comment (pyupew) [inline packages/website/pages/index.tsx]: We'll probably want to use a different image but it's fine if we address this in a future PR

File: packages/website/src/bin/page-generator/index.js [hunk 2]
```diff
@@ -286,18 +287,41 @@ const generateProjectDocsPages = (docsInfo, generatorConfig, name) => {
...
+const generateRootReadMePage = (pagePath, generatorConfig, docKeys) => {
+  if (fs.existsSync(pagePath)) {
+    generateProjectDocPage(
+      'readme.js',
+      pagePath,
+      { key: 'readme' },
+      generatorConfig,
+      'readme',
+    );
+    const result = [{ id: 'packages', pagePath: '/packages' }];
```
Comment (pyupew) [inline packages/website/src/bin/page-generator/index.js]: Can we rename this to something that reflects this being related to packages?

**Pull request merged**